web/src/routes/OrganizationsList/Organization/Tabs/Settings/Settings.tsx

```diff
@@ -95,8 +153,14 @@ const GeneralSettings = (props: GeneralSettingsProps) => {
           justifyContent={{default: 'justifyContentFlexEnd'}}
           width={'100%'}
         >
-          <Button variant="primary">Save</Button>
-          <Button variant="link">Cancel</Button>
+          <Button
+            variant="primary"
+            type="submit"
+            onClick={onSave}
+            isDisabled={!checkForChanges()}
```

Comment:
Small optimization: We may want to put this inside a useEffect that updates a state variable, otherwise this function will be called on every re-render

Reply:
Hmm, we want to call it on every re-render to make sure the button is enabled/disabled based on the changes in the form.